### ISSUE TITLE
KaTeX parse error: Undefined control sequence: \below at position 5:

### ISSUE THREAD

**Issue by shanshouchen** [mathjax/MathJax#2262]
I try to render the formula to web , but I get error.
**formula:**
`\lim\below{n\rightarrow\infty}{\left(1+\frac{1}{n}\right)^n}`
![image](https://user-images.githubusercontent.com/9147956/70707782-f4640080-1d13-11ea-8fda-a28414e4e16a.png)
**error formula:**
![image](https://user-images.githubusercontent.com/9147956/70707800-0180ef80-1d14-11ea-9201-449777a8ad84.png)

**Comment by dpvc** [comment]
The error message in our issue title is not a MathJax error message, but a KaTeX one, so you might need to contact KaTeX rather than MathJax for that.  But the message is indicating that the `\below` macro is not defined.  This is not a standard LaTeX macro, so may be something locally defined from wherever you copied the expression.  You might try replacing it with `_` or perhaps `\limits_`, e.g., 

    \lim\limits_{n\rightarrow\infty}{\left(1+\frac{1}{n}\right)^n}

**Comment by shanshouchen** [comment]
Thank you very much for your answer, which was very useful to me.
This expression is generated by Microsoft Word. So far, is MathJax unable to fully support the formula expressions generated by Microsoft Word? If so, are you all unsupported? Is there any official documentation? Thanks again.

**Comment by shanshouchen** [comment]
> The error message in our issue title is not a MathJax error message, but a KaTeX one, so you might need to contact KaTeX rather than MathJax for that. But the message is indicating that the `\below` macro is not defined. This is not a standard LaTeX macro, so may be something locally defined from wherever you copied the expression. You might try replacing it with `_` or perhaps `\limits_`, e.g.,
> 
> ```
> \lim\limits_{n\rightarrow\infty}{\left(1+\frac{1}{n}\right)^n}
> ```

Thank you very much for your answer, which was very useful to me.
This expression is generated by Microsoft Word. So far, is MathJax unable to fully support the formula expressions generated by Microsoft Word? If so, are you all unsupported? Is there any official documentation? Thanks again.

**Comment by shanshouchen** [comment]
> The error message in our issue title is not a MathJax error message, but a KaTeX one, so you might need to contact KaTeX rather than MathJax for that. But the message is indicating that the `\below` macro is not defined. This is not a standard LaTeX macro, so may be something locally defined from wherever you copied the expression. You might try replacing it with `_` or perhaps `\limits_`, e.g.,
> 
> ```
> \lim\limits_{n\rightarrow\infty}{\left(1+\frac{1}{n}\right)^n}
> ```

The formula expressions are generated by Microsoft Word, what Settings do I need for MathJax to render directly for me?

**Comment by dpvc** [comment]
Again, the error message you reported is from KaTeX, not MathJax, so what MathJax supports may be immaterial to your situation, as it seems you are using KaTeX, not MathJax (@pkra, that was the reason for the upstream label, since it is not about MathJax).

As for Microsoft Word, it may be that it uses a set of macros definitions that define some non-standard macros that is uses for its output.  The macros that MathJax supports are listed in the [documentation](http://docs.mathjax.org/en/latest/input/tex/macros/index.html).  It is possible to include macro definitions in your MathJax configuration if you are the author of the page in question.  E.g.,

```
<script>
MathJax = {
  tex: {
    macros: {
      below: '\\limits_'
    }
  }
}
</script>
<script src="https://cdn.jsdelivr.net/nopm/mathjax@3/es5/tex-chtml.js" id="MathJax-script"></script>
```

should do it, if you are really using MathJax rather than KaTeX.
